src/components/Login.jsx

```diff
@@ -148,19 +213,25 @@ function Login({ setCredentials }) {
       <form id="LoginForm" onSubmit={submit}>
         E-mail Address:
         <br />
-        <input name="Email" type="text" />
+        <input name="email" type="text" defaultValue={credentials ? credentials.email : ""} />
         <br />
         Password:
         <br />
-        <input name="Password" type="password" />
+        <input
+          name="password"
+          type="password"
+          defaultValue={credentials ? credentials.password : ""}
+        />
         <br />
-        <input type="submit" value="Log In or Register" />
+        <input type="submit" value="Log In or Register" />{" "}
+        <button onClick={forgotPassword}>I Forgot my Password</button>
       </form>
     </>
   );
 }
 
 Login.propTypes = {
+  credentials: PropTypes.object,
```

Comment:
Define the shape of the credentials object for better prop validation.

Reply:
Agreed.  That will be done when the database schema is more stable.